### DISCUSSION TITLE
NGINX default page persisting after application deployed successfully

### DISCUSSION THREAD

**Discussion by LeslieNock** [dokku/dokku#5961] [General]
Hey,

I've been trying to get a website running off a dokku droplet with Digitalocean.

I think I'm correctly configuring the dokku domains and keys, and have successfully pushed my main branch to the dokku remote user. Everything looked successful and I'm getting the notification stating the application has been deployed.
![image](https://github.com/dokku/dokku/assets/6913062/330b6222-ab63-4315-8534-8c9157f298cc)

However, when I navigate to the IP (Nameservers are still being applied for the domain), I'm getting the default NGINX page. Not the expected .NET Core template that I'd pushed over.

Any ideas one what I could be doing wrong here? Happy to provide whatever you believe will be useful to assist.

**Comment by LeslieNock** [comment]
Managed to fix this by removing the default file at `/etc/nginx/sites-enabled`

Fingers crossed subsequent deploys don't put me back to a default NGINX page :)

**Reply by Rec0iL99** [reply]
Hi @LeslieNock, how did you complete the installation process without using the web UI installer? Thanks.
